schemas/IDLs/org.gel.models.metrics.avro/1.1.0/ReportedVsGenetic.avdl

```diff
@@ -59,6 +59,10 @@ protocol ReportedVsGenetic {
         */
         string runDate;
         /**
+        Genome assembly
+        */
+        string genomeAssembly;
```

Comment:
For your own sanity I would recommend making this an `enum` and not a `string`.
See for example:
https://github.com/genomicsengland/GelReportModels/blob/8c8ff2b5006ba45d7d62b8e6519c736c12a55382/schemas/IDLs/org.gel.models.cva.avro/0.4.0/Variant.avdl#L11

Reply:
I think it's a good idea. I will make it enum.
Thanks!